Zend/zend_object_handlers.c

```diff
@@ -1308,13 +1376,21 @@ ZEND_API zval *zend_std_get_property_ptr_ptr(zend_object *zobj, zend_string *nam
 					return &EG(error_zval);
 				}
 			}
+			if (UNEXPECTED(zend_lazy_object_must_init(zobj))) {
+				zobj = zend_lazy_object_init(zobj);
+				if (!zobj) {
+					return &EG(error_zval);
+				}
+
+				return zend_std_get_property_ptr_ptr(zobj, name, type, cache_slot);
+			}
 			if (UNEXPECTED(!zobj->properties)) {
 				rebuild_object_properties_internal(zobj);
 			}
 			if (UNEXPECTED(type == BP_VAR_RW || type == BP_VAR_R)) {
 				zend_error(E_WARNING, "Undefined property: %s::$%s", ZSTR_VAL(zobj->ce->name), ZSTR_VAL(name));
 			}
-			retval = zend_hash_add(zobj->properties, name, &EG(uninitialized_zval));
+			retval = zend_hash_add(zend_std_get_properties(zobj), name, &EG(uninitialized_zval));
```

Comment:
Is this change necessary? Object properties are already built, and lazy is handled.